python/cudf/cudf/tests/test_index.py

```diff
@@ -2098,6 +2098,35 @@ def test_get_loc_single_unique_numeric(idx, key, method):
         assert_eq(expected, got)
 
 
+@pytest.mark.parametrize(
+    "idx", [pd.RangeIndex(3, 100, 4)],
+)
+@pytest.mark.parametrize("key", list(range(1, 110, 3)))
+@pytest.mark.parametrize("method", [None, "ffill"])
+def test_get_loc_rangeindex(idx, key, method):
```

Comment:
`tolerance` is not tested? Not saying we have to support it (since the regular version is also unsupported).

Reply:
Yeah let's punt on that for now in the interest of getting this merged before there are more conflicts. I agree that we should test this, I can open issue.